robottelo/performance/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
+
+Part of functionalities of Candlepin are defined in this module
+and have utilities of single register by activation-key, single
+register and attach, single subscription deletion.
+
+"""
+import logging
+import time
+import requests
+
+from robottelo.common import conf, ssh
+from robottelo.common.helpers import get_server_credentials, get_server_url
+from urlparse import urljoin
+
+
+class CandlepinPerfTest(object):
```

Comment:
s/CandlepinPerfTest/Candlepin as you are in the module inside the performance package we can make the class name shorter.